lib/Service/StatusService.php

```diff
@@ -38,7 +38,7 @@ public function __construct(
 
     public function isProperlyConfigured(): bool
     {
-        $cronMode = $this->settings->getSystemValue('backgroundjobs_mode');
+        $cronMode = $this->settings->getAppValue('core', 'backgroundjobs_mode');
```

Comment:
This'll fix the failing test I think

```suggestion
        $cronMode = $this->settings->getAppValue('core', 'backgroundjobs_mode', 'ajax');
```

Reply:
It's already fixed after some tries.

Since it's only tested for `$cronMode === 'cron'` I've decided to not set any default value.